tests/src/3d/testqgs3drendering.cpp

```diff
@@ -1408,5 +1469,155 @@ void TestQgs3DRendering::testFilteredExtrudedPolygons()
+  grayImage = convertDepthImageToGray16Image( depthImage );
+  QVERIFY( renderCheck( "depth_wheel_action_2", grayImage, 550 ) );
+
+  scene->cameraController()->depthBufferCaptured( depthImage );
+
+  QGSCOMPARENEARVECTOR3D( testCam->zoomPoint(), QVector3D( -32.5, 223.5, 184.7 ), 1.0 );
+  QGSCOMPARENEARVECTOR3D( testCam->cameraPose()->centerPoint(), QVector3D( -32.5, 223.5, 184.7 ), 1.0 );
+  QGSCOMPARENEAR( testCam->cameraPose()->distanceFromCenterPoint(), 757.4, 1.0 );
+  QCOMPARE( testCam->cumulatedWheelY(), 0 );
+
+  // Third wheel action
+  QWheelEvent wheelEvent3( midPos, midPos, QPoint( ), QPoint( 0, 480 ),
+                           Qt::NoButton, Qt::NoModifier, Qt::NoScrollPhase,
+                           false, Qt::MouseEventSynthesizedByApplication );
+  testCam->superOnWheel( new Qt3DInput::QWheelEvent( wheelEvent3 ) );
+  qDebug() << "Checking depth after onWheel called (ter)" ;
+  QCOMPARE( testCam->cumulatedWheelY(), wheelEvent3.angleDelta().y() * 5 );
+  QCOMPARE( testCam->cameraBeforeZoom()->viewCenter(), testCam->cameraPose()->centerPoint().toVector3D() );
+
+  depthImage = Qgs3DUtils::captureSceneDepthBuffer( engine, scene );
+  grayImage = convertDepthImageToGray16Image( depthImage );
+  QVERIFY( renderCheck( "depth_wheel_action_3", grayImage, 550 ) );
+
+  scene->cameraController()->depthBufferCaptured( depthImage );
+
+  QGSCOMPARENEARVECTOR3D( testCam->zoomPoint(), QVector3D( -32.4, 222.8, 184.1 ), 1.0 );
+  QGSCOMPARENEARVECTOR3D( testCam->cameraPose()->centerPoint(), QVector3D( -32.4, 222.8, 184.1 ), 1.0 );
+  QGSCOMPARENEAR( testCam->cameraPose()->distanceFromCenterPoint(), 126.4, 0.1 );
+  QCOMPARE( testCam->cumulatedWheelY(), 0 );
+
+  // Fourth wheel action
+  QWheelEvent wheelEvent4( midPos, midPos, QPoint(), QPoint( 0, 120 ),
+                           Qt::NoButton, Qt::NoModifier, Qt::NoScrollPhase,
+                           false, Qt::MouseEventSynthesizedByApplication );
+  testCam->superOnWheel( new Qt3DInput::QWheelEvent( wheelEvent4 ) );
+  qDebug() << "Checking depth after onWheel called (quad)" ;
+  QCOMPARE( testCam->cumulatedWheelY(), wheelEvent4.angleDelta().y() * 5 );
+  QCOMPARE( testCam->cameraBeforeZoom()->viewCenter(), testCam->cameraPose()->centerPoint().toVector3D() );
+
+  depthImage = Qgs3DUtils::captureSceneDepthBuffer( engine, scene );
+  grayImage = convertDepthImageToGray16Image( depthImage );
+  QVERIFY( renderCheck( "depth_wheel_action_4", grayImage, 550 ) );
+
+  scene->cameraController()->depthBufferCaptured( depthImage );
+
+  QGSCOMPARENEARVECTOR3D( testCam->zoomPoint(), QVector3D( -32.3, 221.7, 183.2 ), 1.0 );
+  QGSCOMPARENEARVECTOR3D( testCam->cameraPose()->centerPoint(), QVector3D( -32.3, 221.7, 183.2 ), 1.0 );
+  QGSCOMPARENEAR( testCam->cameraPose()->distanceFromCenterPoint(), 101.1, 0.1 );
+  QCOMPARE( testCam->cumulatedWheelY(), 0 );
+
+  QVector3D diff = scene->cameraController()->camera()->position() - startPos;
+  qDebug() << "Checking camera position";
```

Comment:
remove this line or change it to a proper message.

Reply:
done!